app/src/main/res/values-v21/themes.xml

```diff
@@ -35,6 +35,7 @@
     <style name="AppThemeSemiTransparent" parent="AppThemeLight">
         <item name="android:textColorSecondary">@color/kiss_text_on_semi_light</item>
         <item name="listBackgroundColor">@color/kiss_background_light_transparent</item>
+        <item name="searchBackgroundColor">@color/kiss_background_light_transparent</item>
```

Comment:
do we need to redefine it here? I think it's inherited from value/themes.xml

Reply:
Removing them from here doesn't make the bars semi-transparent on systems with  >API 21.

From what I understand, values set in `AppThemeLight` at `values/themes.xml` are inherited by `AppThemeSemiTransparent` in `values-v21/themes.xml`, but those set in `AppThemeSemiTransparent` at `values/themes.xml` are not inherited.
Similar situation with `AppThemeSemiTransparentDark`.

Reply:
:+1: